ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentService.scala

```diff
@@ -24,13 +25,12 @@ class DeploymentService(processRepository: FetchingProcessRepository[Future],
                         actionRepository: DbProcessActionRepository,
                         scenarioResolver: ScenarioResolver)(implicit val ec: ExecutionContext) {
 
-  def cancelProcess(processId: ProcessIdWithName, comment: Option[String],
-                   performCancel: ProcessIdWithName => Future[Unit])(implicit user: LoggedUser): Future[ProcessActionEntityData] = {
+  def cancelProcess(processId: ProcessIdWithName, deploymentComment: Option[DeploymentComment], performCancel: ProcessIdWithName => Future[Unit])(implicit user: LoggedUser): Future[ProcessActionEntityData] = {
     for {
       _ <- performCancel(processId)
       maybeVersion <- findDeployedVersion(processId)
       version <- processDataExistOrFail(maybeVersion, processId.name.value)
-      result <- actionRepository.markProcessAsCancelled(processId.id, version, comment)
+      result <- actionRepository.markProcessAsCancelled(processId.id, version, deploymentComment.map(_.value))
```

Comment:
Hmm, why not require DeploymementComment at markProcessAsCancelled / markProcessAsDeployed? :) These are connected to deploy?

Reply:
changed it, thx